ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/SourceBuilder.java

```diff
@@ -387,13 +384,10 @@ private static PhysicalSchema getPhysicalSchema(final SourceStep<?> streamSource
    * </pre>
    */
   private static String changelogTopic(
-      final KsqlQueryBuilder queryBuilder,
+      final RuntimeBuildContext queryBuilder,
       final String stateStoreName
   ) {
-    return ReservedInternalTopics.KSQL_INTERNAL_TOPIC_PREFIX
```

Comment:
to make sure that I understand, this line is the "meat" of the PR, right?

Reply:
yep, that's right